apps/web/src/hooks/use-vault-lock.ts

```diff
@@ -0,0 +1,23 @@
+import { EV, EVENTS } from "@notesnook/core";
+import { useEffect, useState } from "react";
+import Vault from "../common/vault";
+
+export default function useVaultLock() {
```

Comment:
Let's rename this to `useVault` and by convention we no longer do `default` exports.